packages/web/package.json

```diff
@@ -33,7 +33,8 @@
     }
   ],
   "license": "MIT",
-  "main": "dist/openjsad.umd.js",
+  "main": "dist/openjscad.umd.js",
+  "module": "dist/openjscad.umd.js",
```

Comment:
what is the status for the support of the 'module' field ? I used to use it a few years back, then it seemed to go out of fashion and now is back again (saw some es module listings using it to get all npm modules with direct `import ` support)

Reply:
Rollup, Webpack and Browserify to an extent use this during bundling.  The "browser" field is the standard one, but will fall back to "main" if it doesn't exist.  "module" tells the major bundlers to use that file instead of attempting to use the src files (which aren't set).

The upshot is that you can use an import like:

```javascript
const openjscad = require('@jscad/web')
```

instead of 

```javascript
const openjscad = require('@jscad/web/dist/openjscad.umd.js')
```

And if we decide to change the default exported file, we can control that without everyone
chaging their import/requires.

"module" will get skipped by node, and the browsers.